src/vs/workbench/parts/html/browser/html.contribution.ts

```diff
@@ -100,3 +100,16 @@ CommandsRegistry.registerCommand('_workbench.previewHtml', function (accessor: S
 		.openEditor(input, { pinned: true }, position)
 		.then(editor => true);
 });
+
+CommandsRegistry.registerCommand('_workbench.htmlPreview.postMessage', (accessor: ServicesAccessor, resource: URI | string, message: any) => {
+	const uri = resource instanceof URI ? resource : URI.parse(resource);
+	const activePreview = accessor.get(IWorkbenchEditorService).getVisibleEditors()
+		.filter(c => c instanceof HtmlPreviewPart)
+		.map(e => e as HtmlPreviewPart)
+		.filter(e => e.model.uri.scheme == uri.scheme && e.model.uri.path === uri.path);
+	if (activePreview.length) {
```

Comment:
You may want to highlight in the description or doc that only the first visible matched htmlPreview document can get the message.